### PR TITLE
Navigation bar styled. Live updating underlined location based on act…

### PR DESCRIPTION
**Pull request by madwirblet**
CSS for the navigation bar. Underlines the active tab on the website.

### CODE REVIEW

File: my-ai-stock-buddy/src/components/Nav/images/StockBuddyLogo.png
Comment (rileycleavenger) [inline my-ai-stock-buddy/src/components/Nav/images/StockBuddyLogo.png:1]: I love this logo, but do you have a higher quality/larger version that you could replace this file with so it appears more high quality in the nav bar?

**Review by rileycleavenger — CHANGES_REQUESTED** [review]
It looks so good!!. Only thing I would say is just replace the logo with a higher quality version of the image if possible.

File: my-ai-stock-buddy/src/components/Nav/images/StockBuddyLogo.png
Comment (madwirblet) [reply my-ai-stock-buddy/src/components/Nav/images/StockBuddyLogo.png:1]: Updated the logo to be high res

**Review by rileycleavenger — APPROVED** [review]
LGTM! Nice work!

**Pull request merged**